Operational-Procedures/Main-Telescope/Refrigeration-System-Pathfinder/refrigeration-system-pathfinder.rst

```diff
@@ -0,0 +1,65 @@
+.. |author| replace:: *Johnny H. Esteves*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Bruno Quint, Kevin Reil*
+
+.. _ComCam-Master-Calibrations-Procedure:
+
+###############################################
+Refrigeration System - Pathfinder
+###############################################
+
+.. _Refigeration-System-Pathfinder-Overview:
+
+Overview
+========
+
+In this page you find information about the design, reviews and procedures of the refrigeration system, pathfinder.
+
+
+.. _Refigeration-System-Pathfinder-Procedures:
+
+Procedures
+=============
+
+Here you find some of the start up procedures of pathfinder. The information is constantly being updated. The safest way of find the procedures is to search on the docushare. 
```

Comment:
Please, make 1 phrase per line, this makes it easier for us to review future changes.

Reply:
Done